src/main/scala/com/fulcrumgenomics/umi/UmiConsensusCaller.scala

```diff
@@ -369,7 +369,7 @@ trait UmiConsensusCaller[ConsensusRead <: SimpleRead] {
   private def simplifyCigar(cigar: Cigar) = {
     import CigarOperator._
     if (cigar.forall(e => e.operator == M || e.operator == I || e.operator == D)) {
-      cigar
+      cigar.coalesce
```

Comment:
Previously, the consensus caller would treat `5M5M` and `10M` as structurally different!